apps/server/lib/lexical/server/project/intelligence.ex

```diff
@@ -216,6 +188,12 @@ defmodule Lexical.Server.Project.Intelligence do
     {:noreply, state}
   end
 
+  @impl GenServer
+  def handle_info(struct_discovered(module: module_name), %State{} = state) do
+    state = State.add_struct_module(state, module_name)
+    {:noreply, state}
```

Comment:
as expected the module names in `app` files are fully qualified with the app name and all so there should be zero names clashing over here ✅ 
e.g. `Elixir.Lexical.RemoteControl.Api` includes `Elixir` and also the app name `Lexical`.

Reply:
👍